engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -533,7 +535,20 @@ private void addFinalPostProcessingNodes(RenderGraph renderGraph) {
         LateBlurNode secondLateBlurNode = new LateBlurNode("secondLateBlurNode", context, firstLateBlurFbo, secondLateBlurFbo);
         renderGraph.addNode(secondLateBlurNode);
 
-        Node finalPostProcessingNode = new FinalPostProcessingNode("finalPostProcessingNode", context);
+        // DependencyConnection finalIn1 = DependencyConnection.createFBOConnection(1, DependencyConnection.Type.INPUT,toneMappingNode.getOutputFBOUri(1));
+        // FboConnection finalIn2 = DependencyConnection.createFBOConnection(2, DependencyConnection.Type.INPUT,secondLateBlurNode.getOutputFBOUri(2));
+        FinalPostProcessingNode finalPostProcessingNode = new FinalPostProcessingNode("finalPostProcessingNode", context/*finalIn1*/);
+        /**As the last attribute - getting output dependencyUri/wholeconnection(can't see why now, but possible) from nodes - how? -
+         *                                                       either type -
+         *                                                          getOutputFBOConnection{ByNameContains}(substr/prefix/suffix..)/
+         *                                                       or getOutputFBOConnection{ByPriority}(#)-fetch output FBO
+         *                                                         - requires List instead of Map, OR adding priority attribute or...
+         *                                                       or getOutputFBOConnection{ByName}(name)*/
+        finalPostProcessingNode.connectFbo(1, toneMappingNode.getOutputFboConnection(1)); //addInputFboConnection(1, toneMappingNode.getOutputFBOUri(1));
+        finalPostProcessingNode.setDependencies(context);
```

Comment:
Connect tone mapping output to final post process fbo input 1
Call rest of finalPostProcessing setup code